openquake/mbt/tests/tools/fault_modeler/test_fault_modeling_utils.py

```diff
@@ -0,0 +1,294 @@
+import json
+import unittest
+import os
+import numpy as np
+
+from openquake.mbt.tools.fault_modeler.fault_modeling_utils import *
```

Comment:
Use explicit list of symbols allow editors to be more efficient.

Reply:
Matteo, I think explicitly importing 20-30 functions is too large a decrease in programmer efficiency and code readability to compensate for the increase in editor efficiency.

Reply:
I usually import the module with a short name, e.g.:

import openquake.mbt.tools.fault_modeler.fault_modeling_utils as fmu

In this way it is easier for the editor to track the various methods.

Reply:
@cossatot we are talking about on-the-fly lintian for undefined symbols, nothing related to the editor efficency, just to catch accidentally inserted unknown symbols or typos.
@klunk386 in the specific case of tests could be acceptable also your solution.

Reply:
Fixed